logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -797,6 +800,17 @@ internal class ConversationDataSource internal constructor(
     override suspend fun updateConversationName(conversationId: ConversationId, conversationName: String, timestamp: String) =
         wrapStorageRequest { conversationDAO.updateConversationName(idMapper.toDaoModel(conversationId), conversationName, timestamp) }
 
+    override suspend fun deleteUserFromConversations(userId: UserId): Either<CoreFailure, Unit> =
```

Comment:
I think it is a better idea if we rewrite this to being performed in one Transactional query, so we access DB just one and succeed if all operations are good as well =)

Reply:
This too:
https://cashapp.github.io/sqldelight/multiplatform_sqlite/grouping_statements/